src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const inputHandle = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364